package.json

```diff
@@ -47,9 +47,10 @@
     "@ngrx/eslint-plugin": "^15.0.0",
     "@types/jasmine": "~4.3.0",
     "@typescript-eslint/eslint-plugin": "5.44.0",
-    "eslint-plugin-simple-import-sort": "^8.0.0",
+    "autoprefixer": "^10.4.13",
```

Comment:
czy ta paczka nie była powiązana z tailwindem?

Reply:
to jest plugin do sortowania importow

Reply:
usunalem ta paczke